cldf/requirements.txt

```diff
@@ -0,0 +1,46 @@
+appdirs==1.4.3
```

Comment:
wow, we do require a lot of packages to do a simple dataset don't we? can we get rid of some of these (@xrotwang ?)

Reply:
I'm not sure why the `pytest` related packages (`py`, `pytest`, possibly some more) are included. The idea is to only list what's actually imported, i.e. present in `sys.modules`.

Reply:
Ah, `pytest` is imported in `cldfbench.commands.check`. Should be imported there conditionally - then we could get away with less packages. But since the typical lexibanker will have `pytest` installed anyway, it will show up in this list.

The alternative to this somewhat inaccurate heuristic would be a whitelist of packages that we do want to list ... But I think I'd prefer something complete - if not minimal - over something less than minimal.

Reply:
Yes, being able to run tests is important. I just wondered if we could shave a handful off easily.